contrib/rpm/buildah.spec

```diff
@@ -0,0 +1,95 @@
+%if 0%{?fedora} || 0%{?rhel} == 6
+%global with_bundled 1
+%global with_debug 0
+%global with_check 1
+%else
+%global with_bundled 0
+%global with_debug 0
+%global with_check 0
+%endif
+
+%if 0%{?with_debug}
+%global _dwz_low_mem_die_limit 0
+%else
+%global debug_package   %{nil}
+%endif
+
+%global provider        github
+%global provider_tld    com
+%global project         projectatomic
+%global repo            buildah
+# https://github.com/kubernetes-incubator/cri-o
+%global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path     %{provider_prefix}
+%global commit         0235b96c9b9b1c656eab50e63a0d50bb23beb8e0
+%global shortcommit    %(c=%{commit}; echo ${c:0:7})
+
+Name:           buildah
+Version:        0.1
```

Comment:
This doesn't jive with the output we get from "buildah --version", which is 0.0.0 until we start overriding it in main.go.